crates/languages/src/css/config.toml

```diff
@@ -9,6 +9,5 @@ brackets = [
     { start = "\"", end = "\"", close = true, newline = false, not_in = ["string", "comment"] },
     { start = "'", end = "'", close = true, newline = false, not_in = ["string", "comment"] },
 ]
-word_characters = ["-"]
```

Comment:
IMO this change is the wrong choice for CSS.

In class selectors the words _are_ delimited by `-`, so selecting the entire class when you just want to select a word is bad.

It also doesn't match VS Code.